docker/build.sh

```diff
@@ -5,22 +5,24 @@ DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
 
 cd "$DIR/.."
 
-for config in Debug Release ; do
-    for proj in Datadog.Trace Datadog.Trace.ClrProfiler.Managed Datadog.Trace.OpenTracing ; do
-        dotnet publish -f netstandard2.0 -c $config src/$proj/$proj.csproj
+for config in Debug ; do
```

Comment:
nit: No need for a `for` loop if `config` will only have one value. Instead, can we use the `buildConfiguration` variable from CI?